src/main/java/frc/robot/subsystems/arm/ArmBase.java

```diff
@@ -6,57 +6,59 @@
 import com.ctre.phoenix.motorcontrol.can.TalonFX;
 
 import edu.wpi.first.math.controller.ArmFeedforward;
-import edu.wpi.first.wpilibj.AnalogPotentiometer;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+import com.ctre.phoenix.sensors.AbsoluteSensorRange;
+import com.ctre.phoenix.sensors.CANCoder;
+import com.ctre.phoenix.sensors.CANCoderConfiguration;
+import com.ctre.phoenix.sensors.SensorInitializationStrategy;
 
 import static frc.robot.Constants.Wiring.ARM_MOTOR_ID_BASE;
+import static frc.robot.Constants.Wiring.BASE_CANCODER_ID;
 import static frc.robot.Constants.Arm.INV_GEAR_RATIO_BASE;
 import static frc.robot.Constants.FALCON_TICKS_PER_REV;
-import static frc.robot.Constants.Wiring.BASE_POTENTIOMETER_PORTNUM;
-import static frc.robot.Constants.Arm.ARM_BASE_POTENTIOMETER_MULT;
-import static frc.robot.Constants.Arm.ARM_BASE_POTENTIOMETER_ADD;
-import static frc.robot.Constants.Arm.kP_BASE;
-import static frc.robot.Constants.Arm.kI_BASE;
-import static frc.robot.Constants.Arm.kD_BASE;
-import static frc.robot.Constants.Arm.kG_BASE;
-import static frc.robot.Constants.Arm.kV_BASE;
-import static frc.robot.Constants.Arm.kS_BASE;
-import static frc.robot.Constants.Arm.kA_BASE;
-import static frc.robot.Constants.Arm.TELEOP_ANGLE_VELOCITY;
+import static frc.robot.Constants.Arm.*;
 
 public class ArmBase extends SubsystemBase {
 
     private final TalonFX             baseMotor;
-    private final AnalogPotentiometer basePotentiometer;
+    //private final CANCoder canCoder;
     private final ArmFeedforward      feedforward;
-    private final double[]            basePos = { 90, 75, 60, 0, 0, 0, 0, 0, 0,
-            0 };
-
+    private final double[]            basePos = {0, 10, 20, 30, 40, 50, 60, 70, 80, 90};
+    private int degrees;
+    
     public ArmBase() {
+        //canCoder = new CANCoder(BASE_CANCODER_ID);
         baseMotor = new TalonFX(ARM_MOTOR_ID_BASE);
-
         baseMotor.configFactoryDefault();
         baseMotor.enableVoltageCompensation(true);
-        SupplyCurrentLimitConfiguration limit = new SupplyCurrentLimitConfiguration(true, 40, 80, 0.5);
+        SupplyCurrentLimitConfiguration limit = new SupplyCurrentLimitConfiguration(true, 20, 25, 0.5);
         baseMotor.configSupplyCurrentLimit(limit);
         baseMotor.config_kP(0, kP_BASE);
         baseMotor.config_kI(0, kI_BASE);
         baseMotor.config_kD(0, kD_BASE);
+        baseMotor.setSelectedSensorPosition(angleToTick(90));
```

Comment:
Note for future reference: until we have a cancoder, we are powering up the arm at 90 degrees and hardcoding that position.